packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -535,29 +535,28 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
     }
   }
 
-  /// This method can be called after the build phase, during the layout of the
-  /// nearest descendant [RenderObjectWidget] of the gesture detector, to filter
-  /// the list of available semantic actions.
+  /// This method can be called outside of the build phase to filter the list of
+  /// available semantic actions.
```

Comment:
it doesn't actually filter the actions immediately right? It just provides input for the computation done in the next frame (and schedules a frame if necessary), without requiring the relevant widget to be built.

Reply:
Yes, clarified.